launcher/sdw_updater_gui/UpdaterApp.py

```diff
@@ -32,10 +32,11 @@ def launch_securedrop_client():
 
 
 class UpdaterApp(QDialog, Ui_UpdaterDialog):
-    def __init__(self, parent=None):
+    def __init__(self, should_skip_netcheck: bool = False, parent=None):
         super(UpdaterApp, self).__init__(parent)
```

Comment:
The default value here isn't _strictly_ necessary, but it preserves the ability to create the Prelight Updater without any parameters, which looks cleaner and is nice for e.g. the `notify` script.